doc/api/connection.md

```diff
@@ -0,0 +1,158 @@
+# Connection
+
+## Class: Connection
+
+### new Connection(transport, server, client)
+
+* transport: `Transport`.
+* server: `Server` — JSTP server instance, used only for server—side parts
+  of connections (optional, but either server or client is required).
+* client: `Client` — JSTP client instance, used only for client—side parts
+  of connections (optional, but either server or client is required).
+
+Don't call this constructor manually unless you use custom tranport.
+Recommended approach is to call `connect()` function provided by these modules:
+* `net`
```

Comment:
I think it's better to put md links here to the function descriptions even though there is no documentation for them right now to avoid missing them later on. See [this](https://gist.github.com/asabaylus/3071099) and [this](https://github.com/blog/1395-relative-links-in-markup-files).
Also, maybe you will add stubs for them?